buildProjects.sh

```diff
@@ -12,4 +12,16 @@ echo $result
 if [ $result -ne 0 ]; then
     echo "Not all tests passed... Retrying."
     dotnet test -f netcoreapp3.1 Eyes.Sdk.DotNet_Travis.sln --filter $1
-fi
\ No newline at end of file
+fi
+
+echo "running tutorials"
+chmod +x ./tutorials/run.sh
+chmod +x ./tutorials/build.sh
+chmod +x ./tutorials/report.sh
+./tutorials/run.sh
+result=$?
+echo $result
+if [ $result -ne 0 ]; then
+    echo "Not all tutorials passed, the build have failed"
+    exit 1
+fi
```

Comment:
This will make sure that if the build fails but the tutorials pass, the build will pass.
Please create another file, like "buildTutorials.sh" and add it to the `.travis.yml` instead.

Reply:
Added a new "buildTutorials.sh" and updated .travis.yml with it.